conversation.go

```diff
@@ -85,6 +86,17 @@ func (c GoogleGroupConversation) GetAllMessages(client http.Client, removeGmailQ
 		if !ok || len(singleMsgArray0) < 9 {
 			continue
 		}
+		singleMsgArray2, ok := singleMsgArray[2].([]interface{})
+		if ok && len(singleMsgArray2) > 0 {
+			for _, singleFileArray := range singleMsgArray2 {
+				singleFile := singleFileArray.([]interface{})
```

Comment:
Please make sure that the array `singleFile` is longer that 5 by using an if statement.

Reply:
@Kininaru that->than

Reply:
@Zxilly Thanks, corrected.

Reply:
@Kininaru Fixed